charts/persistence/README.md

```diff
@@ -1,19 +1,31 @@
 # persistence
 
-![Version: 1.0.3](https://img.shields.io/badge/Version-1.0.3-informational?style=flat-square)
+![Version: 1.0.4](https://img.shields.io/badge/Version-1.0.4-informational?style=flat-square)
 
 Data persistence for UrbanOS using Trino and the Hive Metastore
 
+## Source Code
+
+* <https://github.com/trinodb/trino>
+* <https://github.com/minio/operator>
+
+## Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://operator.min.io/ | minio-operator | 4.5.3 |
+| https://trinodb.github.io/charts/ | trino | 0.8.0 |
+
 ## Values
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
+| global.buckets.hiveStorageBucket | string | `"presto-hive-storage"` |  |
 | global.objectStore.accessKey | string | `"example-key"` |  |
-| global.objectStore.hiveStorageBucket | string | `"presto-hive-storage"` |  |
+| global.objectStore.accessSecret | string | `"example-secret"` |  |
```

Comment:
Does this need to be set somewhere?

Reply:
If I remember correctly, this is a placeholder for secrets to populate it

Reply:
There are values described in the base chart values too to make it clear this needs to be set 👍 
https://github.com/UrbanOS-Public/charts/blob/f432ea7e674ddf5c5ada8c1832662826ea4c99b9/charts/persistence/values.yaml#L7

Docs were just never updated

Reply:
That's what I would expect as well, from the kubernetes secrets right? Is setting these values on the clusters part of this card?

Reply:
setting those values are not part of this card